### ISSUE TITLE
[UX] Visual Design needed for incorporation of Writing Program into Internship Programs page

### ISSUE THREAD

**Issue by leecalcote** [layer5io/layer5#4918]
See https://github.com/layer5io/layer5/issues/4917

---
<img src="https://raw.githubusercontent.com/layer5io/layer5/master/.github/assets/images/layer5/5-light-small.svg" width="24px" align="left" /><h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
-  🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

<img src="https://raw.githubusercontent.com/layer5io/layer5/master/.github/assets/images/buttons/community.webp" height="22px" align="left" />Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

**Comment by aman44444** [comment]
@leecalcote I want to work on this issue .Can you please provide me visual design of Writing Program section?

**Comment by rcypankaj** [comment]
Hlo Is it open, if yes pls assigned it to me, i like to work on this issue.

**Comment by kartikaysaxena** [comment]
Can I work on it?

**Comment by abhijeetgauravm** [comment]
Assigning in chronological order, there u go @rcypankaj

**Comment by shubhankar-mern** [comment]
if its open i would like to work on it

**Comment by abhijeetgauravm** [comment]
@rcypankaj any updates here?

**Comment by rcypankaj** [comment]
Mr. Sir, I'm working on other projects for a firm and don't have time to
finish the layers task. You may assign this work to anyone who can do it
before me..
Please accept our apologies for the inconvenience.

**Comment by ptdatta** [comment]
@leecalcote @Mohith234 I want to take it

**Comment by abhijeetgauravm** [comment]
@kartikaysaxena @shubhankar-mern do u want to work on this issue still?

**Comment by kartikaysaxena** [comment]
> @kartikaysaxena @shubhankar-mern do you still want to work on this issue?

Yes I want to do this.

**Comment by shubhankar-mern** [comment]
Yes

**Comment by shubhankar-mern** [comment]
> > @kartikaysaxena @shubhankar-mern do you still want to work on this issue?
> 
> Yes I want to do this.

Yes

**Comment by abhijeetgauravm** [comment]
Thanks for volunteering gents. Assigning in chronological order and @shubhankar-mern we do have a bunch of good first issues open you are most welcome to contribute!

**Comment by abhijeetgauravm** [comment]
@kartikaysaxena any updates here?

**Comment by abhijeetgauravm** [comment]
Since there is no updates from @kartikaysaxena . Let it get assigned to u @shubhankar-mern !

**Comment by shubhankar-mern** [comment]
@abhijeetgauravm thanks will look into the issue

**Comment by abhijeetgauravm** [comment]
@ptdatta updates here?

**Comment by Affanzuhaib** [comment]
Any update @ptdatta?
Are you working on this.

**Comment by ptdatta** [comment]
> Any update @ptdatta? Are you working on this?

I'm really busy with other issues. If you like you can take it.

**Comment by ankushbhardwaj408** [comment]
@vishalvivekm I think this issue needs to be completed before #4917 !!

**Comment by ankushbhardwaj408** [comment]
![writing](https://github.com/layer5io/layer5/assets/93874580/db6a13d7-9952-4bbd-9b3c-65668d65cb92)
@abhijeetgauravm please take a look at that.

**Comment by vishalvivekm** [comment]
> @vishalvivekm I think this issue needs to be completed before #4917 !!

@ankushbhardwaj408 Yes a mockup will be needed before a potential implementation.

**Comment by vishalvivekm** [comment]
> > @vishalvivekm I think this issue needs to be completed before #4917 !!
> 
> @ankushbhardwaj408 Yes a mockup will be needed before a potential implementation. Would you like to take it up?

**Comment by ankushbhardwaj408** [comment]
@vishalvivekm Actually I am not  familiar with UI/UX design but please check it out.

> ![writing](https://user-images.githubusercontent.com/93874580/277551523-db6a13d7-9952-4bbd-9b3c-65668d65cb92.png) @abhijeetgauravm please take a look at that.

**Comment by ankushbhardwaj408** [comment]
![WhatsApp_Image_2023-10-07_at_23 17 36_eef6c5ea-removebg-preview](https://github.com/layer5io/layer5/assets/93874580/ec0722e4-bf61-47f8-8fb7-711142f78635)

**Comment by abhijeetgauravm** [comment]
> @vishalvivekm Actually I am not familiar with UI/UX design but i'll try my best and please check this out.
> 
> > ![writing](https://user-images.githubusercontent.com/93874580/277551523-db6a13d7-9952-4bbd-9b3c-65668d65cb92.png) @abhijeetgauravm please take a look at that.

Actually We will need mockup for how writing page will be incorporated into internship program page.  // @vishalvivekm @Mohith234 @ritiksaxena124

**Comment by Akshun-01** [comment]
Hey, I can give it a try. If it's still open and unresolved. @Mohith234 @vishalvivekm

**Comment by Akshun-01** [comment]
Okay so after analyzing the issue, I observed that both the pages have very different layout and structuring.
So, do we want to restructure the pages to merge into one and they look like a single page or are we planning to just merge them under lets say 'programs' page and from there we further give link to these two seperate pages?

**Comment by Akshun-01** [comment]
- Like  in the second approach what we can do is make a page ("programs" for example) and then their we brief about both internship and writing programs and give a hyperlink to the already existing pages.
- Otherwise we would need to redesign the programs page, to incorporate both.

So, can anyone clarify what we are trying to achieve?

**Comment by abhijeetgauravm** [comment]
Hey @Akshun-01  Let's discuss this on website call tomorrow at 5:30 PM IST / 7 AM Central time. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.lohhtewfwima).

**Comment by saurabh100ni** [comment]
any updates @Akshun-01 ?

**Comment by saurabh100ni** [comment]
@Akshun-01 did you checkout [Brand-kit](https://layer5.io/company/brand) and [Figma](https://www.figma.com/files/team/972159062361165490/Layer5?fuid=985940838624039218) to get right inspiration for this ?

**Comment by Akshun-01** [comment]
@saurabh100ni, Hey I was busy with some work. I'll share a draft by Saturday. Also, regarding the inspiration, I don't have access to the figma file. So please see if you can help with that.

**Comment by saurabh100ni** [comment]
@Akshun-01 read access is fine to take inspiration through the designs for this one, although you may have it in your way someday 👀, till then you can use your personal ones

**Comment by Akshun-01** [comment]
Hey, I came up with a draft for the writing programs card to be included in the Internship Program.
I would really appreciate some feedback and reviews on this. 
@leecalcote @saurabh100ni @ritiksaxena124 
![writingProgram](https://github.com/layer5io/layer5/assets/93793691/0712c8b6-cf51-40a8-ba76-352bcbd713b0)

**Comment by ritiksaxena124** [comment]
The pen you are using on the card is not smooth from the edges.  You can use the same writer's program logo that we already have - https://layer5.io/community/handbook/recognition
![image](https://github.com/layer5io/layer5/assets/51539117/ef8a3cbb-ec46-4dc6-8bc0-8ebcfbf25971)

**Comment by ritiksaxena124** [comment]
Hey @Akshun-01, The Writing Program is not an internship on its own so it should not be listed as its own card but can be listed on other pages so that people are aware of the program. One possible place can be https://layer5.io/careers/internships, maybe, as it is a program by Layer5.

**Comment by aman44444** [comment]
is it open for everyone? can i make draft PR here?

**Comment by sandramsc** [comment]
> is it open for everyone? can i make draft PR here?

Hi @aman44444 if your question above is regarding opening a PR, issues are open for every contributor:

- read the issue description
- ask for the issue to be assigned to you as this is done on a first come first serve basis (with a proposed solution), consider taking a look at the ["How to contribute"](https://layer5.io/community/newcomers) portion on our newcomers website.
- only after the issue has been assigned are you encouraged to create a PR for it.
- if you find an issue that you are particularly eager to work on and it has already been assigned, consider requesting to collaborate with the assignee.

**Comment by sandramsc** [comment]
This issue has been open for some time with no recent activity, unassigning to open it up for new contributors to give it a go.
